tests/function-controller/cmd/main.go

```diff
@@ -54,20 +55,20 @@ func loadRestConfigWithContext(apiServerURL string, loader clientcmd.ClientConfi
 		}).ClientConfig()
 }
 
-type scenario struct {
-	displayName string
-	scenario    testScenario
+type testSuite struct {
+	name string
+	test test
 }
 
-var availableScenarios = map[string][]scenario{
+var availableScenarios = map[string][]testSuite{
```

Comment:
Shouldn't this variable be sth like `availableTests`? Cuz you get rid of "scenario" names in this file.